src/interfaces/vault.ts

```diff
@@ -44,9 +44,17 @@ export class VaultInterface<T extends ChainId> extends ServiceInterface<T> {
     const cached = await this.cachedFetcherGet.fetch();
     if (cached) {
       if (addresses) {
-        return cached.filter(vault => addresses.includes(vault.address));
+        return cached
```

Comment:
We don't need to change the logic about the data fetched from the caching service since it uses the sdk itself, so once this PR has been merged and released, we can upgrade the sdk version that the caching service uses and `this.cachedFetcherGet.fetch()` will return vaults with the changes from this PR